bd_tools/__main__.py

```diff
@@ -0,0 +1,58 @@
+"""The betz drive tools package."""
+import argparse
+import sys
+from pathlib import Path
+
+BIN_PATH = Path(__file__).parent / "bin"
+
+
+def get_tools():
+    """Returns all scripts in the bin directory"""
+    tools = []
+    for tool in BIN_PATH.glob("*.py"):
+        tools.append(tool.stem)
+    return tools
```

Comment:
maybe more succinct as: `list(BIN_PATH.glob("*.py"))`

Reply:
oh, actually I just noticed the `.stem` sorry, maybe `[p.stem for p in BIN_PATH.glob("*.py")` or `list(map(lambda p: p.stem, BIN_PATH.glob("*.py")))`?